README.md

```diff
@@ -80,6 +86,12 @@ This code can automatically convert an OpenSim model to the external function us
 
 A [full overview of settings](./Documentation/SettingsOverview.md) is available in the documentation.
 
+## Replicate results
+
+Results of previous publications can be replicated by specifying one of the inputs for [initializeSettings](main.m#L18):
```

Comment:
Also need to set subject name and osim_path

Reply:
osim_path should be fine if subject name is ok, no?